modules/store/src/connextStore.ts

```diff
@@ -60,8 +63,24 @@ export class ConnextStore implements IClientStore {
           ),
         );
         break;
+      }
 
-      case StoreTypes.File:
+      case StoreTypes.Postgres: {
+        this.internalStore = new KeyValueStorage(
+          (opts.storage as WrappedPostgresStorage) ||
+            new WrappedPostgresStorage(
+              this.prefix,
+              this.separator,
+              DEFAULT_POSTGRES_TABLE_NAME,
+              opts.sequelize,
+              opts.postgresConnectionUri,
```

Comment:
client stores usually have defaults -- are we ignoring those with the postgres one? If so, it should probably error out at some point before trying to create the store

Reply:
It does error in the wrapped version.